scripts/aclshow

```diff
@@ -124,8 +123,19 @@ class AclStat(object):
             """
             Get ACL counters from the DB
             """
+            counters_db_separator = self.db.get_db_separator(self.db.COUNTERS_DB)
+            rule_to_counter_map = self.db.get_all(self.db.COUNTERS_DB, ACL_COUNTER_RULE_MAP)
             for table, rule in self.acl_rules:
-                cnt_props = lowercase_keys(self.db.get_all(self.db.COUNTERS_DB, "COUNTERS:%s:%s" % (table, rule)))
+                rule_identifier = table + counters_db_separator + rule
+                if not rule_to_counter_map:
+                    self.acl_counters[table, rule] = {}
+                    continue
```

Comment:
@stepanblyschak doesn't make sense when `not rule_to_counter_map`

Reply:
need to create an empty entry in self.acl_counter

Reply:
@stepanblyschak just do it once before all the checks to avoid code duplication

Reply:
So it looks like the suggestion is defaultdict

Reply:
Yes, please check updated code